tests/cgroupstats_tests.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from gnlpy.cgroupstats import CgroupstatsClient
+
+import re
+import traceback
+import unittest
+
+
+CGROUP_PATH = '/proc/self/cgroup'
+CGROUP_CPU_PATH = '/sys/fs/cgroup/cpu/'
+
+
+class CgroupstatsTestCase(unittest.TestCase):
+
+    def setUp(self):
+        # check for presence of cpu cgroup so we know what to assert for
+        try:
+            with open(CGROUP_PATH, mode='r') as f:
+                file_contents = f.read()
+                if re.search('[,:]cpu[,:]', file_contents) is None:
+                    self.cgroup_available = False
+                else:
+                    self.cgroup_available = True
+        except IOError:
+            self.cgroup_available = False
+
+    def test_get_cgroup_stats(self):
```

Comment:
@chantra Yeah that makes sense. I'll get rid of that try/except on the setUp. However we can still run into the case in which the cgroup cpu subsystem is not available (it's not on the supplied Vagrant test box, for instance). If that is the case do you think we should skip the test or use the `exceptedFailure` decorator? If we were to use `skipIf` decorator (well, using our own decorator that would pass in the value of a variable such as `cpu_cgroup_present`) we can pass a reason, which I assume will be part of the test run output.